### DISCUSSION TITLE
Just a thought... Adding a Fork of DockPanelSuite?

### DISCUSSION THREAD

**Discussion by tajbender** [dahall/Vanara#236] [Ideas]
Dear Mr. @dahall,

you at least have heard of me, I'm the guy over from Germany who develops this https://github.com/electrifier/electrifier.

Perhaps you even heard of the famous @DockPanelSuite. As a Windows Insider I recently grabbed an official copy of Windows 11.

I had some ideas for my design, so I was playing around with @electrifier, your @dahall stuff, @DockPanelSuite and last but not least @harborsiem's famous and native https://github.com/harborsiem/WindowsRibbon and came to the following result just after a few hours:

![electrifier main window 07.2.21](https://github.com/electrifier/electrifier/blob/master/docs/Pictures/electrifier%20Main%20Window,%207.2.21.WIP.png?raw=true)

My idea now is to have this fork I created as an base for the development that allows such design features as a C# control, that is distributed as part of the Vanara Library, while I learn some skills and adopt this mushup into a stable open source control.

The only thing I'd like is to have some answers how to get my c# skills to the next level, help with alpha-blending, C#5/C#6, quality control, automated testing, etc. pp. Would be nice if we could get in touch personally via chat or e-mail.

Have a nice weekend, and thanks in advance, regards,

tajbender

**Comment by tajbender** [comment]
Already up and running:

![electrifier Main Window, 7 2 21 WIP#2](https://user-images.githubusercontent.com/10961499/124330139-15f23a80-db8d-11eb-8e2e-9f651d627942.png)

**Comment by dahall** [comment]
Why create a fork for DockPanelSuite? Is it no longer being maintained? If not, have you asked the authors if they would allow you to take ownership? I personally think it is counterproductive to fork a well-known project as you lose all the other contributors, PRs, issues, etc.

**Reply by tajbender** [reply]
> Why create a fork for DockPanelSuite? Is it no longer being maintained? If not, have you asked the authors if they would allow you to take ownership? I personally think it is counterproductive to fork a well-known project as you lose all the other contributors, PRs, issues, etc.

To have your question finally answered: *YES*, they would allow you to take ownership

**Comment by tajbender** [comment]
There's already a repository of https://github.com/tajbender/DockPanelsEleven

I love the name 🍼
